app/controllers/plant_lines_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PlantLinesController < ApplicationController
+  def index
+  end
+
+  def data_grid
```

Comment:
I would use index action with response depending on request Accept header for more RESTful API.